lib/mirage.ml

```diff
@@ -1814,19 +1757,21 @@ module Project = struct
       ]
 
       method packages =
-        let l = [ "lwt"; "mirage-types"; "mirage-types-lwt" ] in
-        Key.match_ Key.(value target) @@ function
-        | `Xen | `Qubes -> "mirage-xen" :: l
-        | `Virtio -> "solo5-kernel-virtio" :: "mirage-solo5" :: l
-        | `Ukvm -> "solo5-kernel-ukvm" :: "mirage-solo5" :: l
-        | `Unix | `MacOSX -> "mirage-unix" :: l
-
-      method libraries =
-        let l = [ "mirage-runtime"; "mirage-types"; "mirage-types.lwt" ] in
+        let l = [
+          (* XXX: use %%VERSION_NUM%% here instead of hardcoding a version? *)
+          package "lwt";
+          package ~ocamlfind:[] "mirage-types-lwt";
+          package ~sublibs:["lwt"] ~min:"3.0.0"  "mirage-types";
```

Comment:
yes. no.  soon to be changed... atm there is mirage-types which provides V1 and V1_LWT in its lwt subpackage, and mirage-types-lwt is a virtual package which gathers all deps to get mirage-types.lwt.  pretty insane atm, should be better soon...